app/views/handlers/list_repeating_question.py

```diff
@@ -67,7 +67,10 @@ def get_previous_location_url(self) -> str:
     def handle_post(self) -> None:
         self.questionnaire_store_updater.add_completed_location(self.current_location)
         if not self.get_first_incomplete_list_repeating_block_location_for_list_item(
-            repeating_block_ids=self.repeating_block_ids,
+            repeating_block_ids=[
```

Comment:
this change is going to be needed for the `get_previous_location_url` as well, currently it will use the index of the repeating block against all repeating blocks, not just those for the list - so I reckon it should be a reusable property, see comment on list edit question